x-pack/plugins/security_solution_serverless/public/get_started/welcome_panel/index.test.tsx

```diff
@@ -7,26 +7,60 @@
 
 import React from 'react';
 import { render, type RenderResult } from '@testing-library/react';
-import { WelcomePanel } from './welcome_panel';
+import { NavigationProvider } from '@kbn/security-solution-navigation';
```

Comment:
We can just use the navigation context mock:
```
jest.mock('@kbn/security-solution-navigation/src/context');
```
It sets a mocked context, so we don't need to add the `NavigationProvider` wrapper in the test, the context will always exist.